lib/matplotlib/tests/test_axes.py

```diff
@@ -4917,6 +4917,22 @@ def test_dash_offset():
         ax.plot(x, j*y, ls=(j, (10, 10)), lw=5, color='k')
 
 
+def test_title_pad():
+    # check that title padding puts the title in the right
+    # place...
+    fig, ax = plt.subplots()
+    ax.set_title('aardvark', pad=30.)
```

Comment:
:+1: for use of 'aardvark'!

Reply:
Wish I could claim credit, but I just cut and pasted from elsewhere.  I hope its a good inside joke...

Reply:
Well at least it is likely to be the first word in English-dictionaries, isn't it ^^? It may actually be(come) a running gag in Matploltib code too: I used it as well in some tests with strings, copy-pasting it from some other existing code section as @jklymak did... (I did not even understand it was a proper word at that time 🐑 )